pages/03.Tutorials/23.using-service-accounts-with-rbac/default.en.md

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Using service accounts with rbac roles'
+published: true
+taxonomy:
+    tag:
+        - kubernetes
+        - service accounts
+        - rbac
+        - user management
+---
+
+The default MetaKube Cluster comes with one admin account to use.
+
+## Prerequisites
+
+* You need to have [created a MetaKube cluster](../02.create-a-cluster/default.en.md) and [installed and configured kubectl](../07.using-kubectl/default.en.md).
+
+## Creating your first view only service account
+
+```shell
+$ kubectl create serviceaccount metakube
+serviceaccount/metakube created
+```
+
+## Fetch the service account with the service token secret
```

Comment:
The next 3 steps could be all handled with one command, of course the command would be a bit more "magic", unsure if this makes it easier to follow or not.